src/util/AuthUtils.js

```diff
@@ -172,11 +172,14 @@ export default class AuthUtils {
 
   /**
    * Check whether the user has a particular permission.
-   * TO BE IMPLEMENTED
    */
-  static hasPermission(/* user: any, permission: string */): boolean {
+  static hasPermission(user: any, permission: string): boolean {
     if (AuthUtils.config && AuthUtils.config.ALL && AuthUtils.config.ALL.authType === 'NONE') {
```

Comment:
Why are we not doing the check when the authType is set to "NONE"? This is the case when Search UI is being run as a module on the node instead of with the servlet.